Tests/SonarScanner.MSBuild.Test/AnalysisWarning/WarningsSerializerTest.cs

```diff
@@ -41,7 +41,7 @@ public class WarningsSerializerTest
 
         [TestMethod]
         public void SerializeToFile_EmptyFileDirectory_Throws() =>
-            ((Action)(() => WarningsSerializer.Serialize(Array.Empty<Warning>(), string.Empty))).Should().Throw<ArgumentException>().WithMessage("Empty path name is not legal.");
+            ((Action)(() => WarningsSerializer.Serialize(Array.Empty<Warning>(), string.Empty))).Should().Throw<ArgumentException>().And.Message.Should().Contain("Empty path name is not legal.");
```

Comment:
Not sure if the issue is also with a prefix.
```suggestion
            ((Action)(() => WarningsSerializer.Serialize(Array.Empty<Warning>(), string.Empty))).Should().Throw<ArgumentException>().WithMessage("Empty path name is not legal.*");
```